PROJECTS.md

```diff
@@ -0,0 +1,47 @@
+# Projects
+
+## Table of Contents
+
+- [Overview](#overview)
+- [Project Facts](#project-facts)
+
+## Overview
+
+Eiffel Community is home to various projects that enable the members of the community to use, contribute,
+and collaborate on Eiffel.
+
+The projects that are part of Eiffel Community adhere to the processes and procedured set by the community
+and follow the [project lifecycle process](./PROJECT_LIFECYCLE.md) for their creation, progression, and
+archival.
+
+Projects hosted in Eiffel Community could utilize one or more repositories under Eiffel Community organization
+on GitHub and the main project repository is where the project facts are documented.
+
+This document lists existing projects, links to the main project repository, and the lifecycle stage the
+projects currently reside in.
+
+Any change to the project facts listed in this document shall be done according to the instructions documented
+in [project lifecycle process](./PROJECT_LIFECYCLE.md).
+
+## Project Facts
+
+The table below lists the projects hosted by Eiffel Community alphabetically. (except the protocol itself)
```

Comment:
Does the the following wording make it more clear that "except the protocol itself" is about the sorting?

```suggestion
The table below lists the projects hosted by Eiffel Community alphabetically with the protocol itself listed first.
```